.github/workflows/devnet_deploy_contracts.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy devnet contracts
+
+on: workflow_dispatch
+
+jobs:
+  contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          context: app
+
+      - name: Install Doppler CLI
+        uses: dopplerhq/cli-action@v1
+
+      - name: Deploy contracts
+        id: deploy-contracts
+        env:
+          DOPPLER_TOKEN: ${{ secrets.DEVNET_DOPPLER_TOKEN_CONTRACTS_DEPLOY }}
+        run: |
+          cd hardhat
+          npm ci
+            doppler run -- npx hardhat deploy --network devnet
+        
+      - name: Commit and push
+        run: |
+          git add .
```

Comment:
Optional, but might be good to target the directory where we expect changes. Helps to document it a bit better.

Reply:
We may need to generate binding, which is not in hardhat directory

Reply:
I think @bgins means: `git add pkg/web3/bindings`

Reply:
Have you identified which files are going to be added here?
Can you list them?

Reply:
Folders:
    * pkg/web3/bindings
    * hardhat/deployments
    * hardhat/.openzeppelin

Reply:
The you have your list for `git add`